packages/plugins/plugin-navtree/src/components/NavTree/NavTree.stories.tsx

```diff
@@ -36,13 +38,54 @@ faker.seed(1234);
 
 const StoryState = defineCapability<{ tab: string }>('story-state');
 
+// TODO(burdon): How to adjust existing surfaces for this test?
+// TODO(burdon): How to toggle attention in this test?
+// TODO(burdon): Fix outline (e.g., button in sidebar nav is clipped when focused).
+// TODO(burdon): Consider similar containment of: Table, Sheet, Kanban Column, Form, etc.
+
+const container = 'flex flex-col grow gap-2 p-4 rounded-md';
```

Comment:
I would still assert `flex flex-col` is not a good default, it should only apply where it is needed.

I think it’s actually the Planks themselves, rather than their content, which might benefit most from the encapsulation here, similar to Fibery’s approach.